DEVELOPMENT.md

```diff
@@ -808,7 +804,36 @@ We use [Cypress](https://cypress.io) for running frontend-based end-to-end tests
 
 #### Running frontend end-to-end tests locally
 
-Make sure to [build the frontend assets](#building-the-frontend) and run `tools/bin/mage dev:initStack dev:sqlDump` to create a seed database which the tests can reset the database to in between runs. To run the stack when working on end-to-end tests, use the `tools/bin/mage dev:startDevStack` command. This will run the runs The Things Stack with proper configuration for the end-to-end tests. Note: this command does not output anything, but logs are written to `.cache/devStack.log`.
+The preferred way to run the end-to-end tests is to use the interactive launcher tool with:
+
+```bash
+$  tools/bin/mage dev:serveDevWebui
```

Comment:
Can you explain what the user should expect here?

It starts the stack and also does some webpack stuff. But that webpack stuff can take a while it seems. Does the developer have to wait for something to be ready?

Reply:
For example after some time I get:

```
[js:serve] Executing additional scripts before exit
[js:serve] 89 assets
3662 modules
webpack 5.96.1 compiled successfully in 18516 ms
```

Is this "ready" ?

Reply:
This means ready indeed

Reply:
seeing:
```
[js:serve] <i> [webpack-dev-server] Project is running at:
[js:serve] <i> [webpack-dev-server] Loopback: http://localhost:8080/, http://[::1]:8080/
```
and later 
```
[stack] DEBUG	Creating listener	{"address": ":8887"}
[stack] DEBUG	Creating listener	{"address": ":1889"}
DEBUG	Creating listener	{"address": ":1887"}
DEBUG	Creating listener	{"address": ":8881"}
INFO	New patch version available	{"current": "3.32.2-dev", "docs_url": "https://www.thethingsindustries.com/docs/getting-started/upgrading/", "latest": "3.32.3"}
DEBUG	Creating listener	{"address": ":8883"}
DEBUG	Creating listener	{"address": ":8889"}
DEBUG	Creating listener	{"address": ":1881"}
DEBUG	Creating listener	{"address": ":8882"}
DEBUG	Creating listener	{"address": ":1883"}
DEBUG	Creating listener	{"address": ":1882"}
```
means ready

Reply:
I've updated `DEVELOPMENT.md` with this